src/internal.c

```diff
@@ -1636,6 +1636,123 @@ int InitSSL_Side(WOLFSSL* ssl, word16 side)
 }
 #endif /* OPENSSL_EXTRA || WOLFSSL_EITHER_SIDE */
 
+#if defined(HAVE_SESSION_TICKET) && !defined(NO_WOLFSSL_SERVER) && \
```

Comment:
This will be replaced by the new implementation.

Reply:
Should I remove this or wait for your implementation?

Reply:
Remove this now and after my callback implementation is merged then I'll approve this pull request.

Reply:
Removed.